apps/generic_terminal/unix_utilities.py

```diff
@@ -0,0 +1,90 @@
+from talon import Context, Module
+
+from ...code.user_settings import get_list_from_csv
+
+ctx = Context()
+mod = Module()
+ctx.matches = r"""
+tag: terminal
+and tag: user.unix_utilities
```

Comment:
I'm not sure if it makes sense to use the tag in the file where it is declared. I would like advice here.

Reply:
I think you'd just make a context without a matches block. This is the least specific match and so would let anybody override it if they wanted.

Reply:
I adjusted this in the current iteration. Thanks for the suggestion :)